integration/tests/bft/gossipsub-signature/signature-requirement/gossipsub_signature_requirement_test.go

```diff
@@ -0,0 +1,39 @@
+package signature_requirement
```

Comment:
Suggestion: renaming this directory (and package) to `integration/tests/bft/gossipsub/signature`.

Reply:
https://github.com/onflow/flow-go/pull/3689/commits/722440a26e5a2d720d3380cedab84abe9808e809